app/assets/src/components/views/PipelineViz.jsx

```diff
@@ -27,18 +28,19 @@ class PipelineViz extends React.Component {
 
     this.state = {
       stagesOpened: [true, true, true, true],
-      zoom: 1,
     };
   }
 
   componentDidMount() {
     this.drawGraphs();
   }
 
-  handleMouseWheelZoom = e => {
-    const { zoomChangeInterval } = this.props;
-    const zoomChange = (e.deltaY < 0 ? 1 : -1) * zoomChangeInterval;
-    this.setState({ zoom: this.state.zoom + zoomChange });
+  zoomIn = () => {
```

Comment:
these could just be `zoomIn = this.panZoomContainer.zoomIn;`

Reply:
I tried that (I think? Unless I am doing something wrong syntactically) but it errors thinking that `this.panZoomContainer` is undefined

Reply:
I can modify the `on[Plus/Minus]Click` prop to `() => this.panZoomContainer.zoom[In/Out]` though

Reply:
oh it probably is undefined at that point. nvm then

Reply:
Why not just use this last suggestion directly . on the component below?